.app/create_post.sh

```diff
@@ -32,7 +32,15 @@ echo -n "Category:     "
 read -er category
 echo -n "Tags (comma separated words or phrases): "
 read -er tags
-# if -all given, ask for all meta info
+
```

Comment:
if `-all` not given, the vars below were not being set (index pages, for example)